src/DIRAC/Core/Utilities/Glue2.py

```diff
@@ -24,19 +24,14 @@
 from DIRAC.ConfigurationSystem.Client.Helpers.Resources import getCESiteMapping, getGOCSiteName
 from DIRAC.Core.Utilities.ReturnValues import S_OK, S_ERROR
 from DIRAC.Core.Utilities.List import breakListIntoChunks
+from DIRAC.Core.Utilities.Grid import ldapsearchBDII
 
 __RCSID__ = "$Id$"
 
 
 sLog = gLogger.getSubLogger(__name__)
 
 
-def __ldapsearchBDII(*args, **kwargs):
```

Comment:
That won't be needed anymore?

Reply:
Now we can directly use ldapsearchBDII from Grid.py, no more circular import